react/features/mobile/navigation/screenOptions.ts

```diff
@@ -244,6 +244,7 @@ export const lobbyNavigationContainerScreenOptions = {
  * Screen options for settings navigation container screen.
  */
 export const settingsNavigationContainerScreenOptions = {
+    ...modalPresentation,
```

Comment:
We don't present it modally on the welcome page. Does this break anything there?

Should we have 2 screens wrapping the same component with different settings?

Reply:
These screen options are applied only on the ConferenceNavigationContainer.

Reply:
https://github.com/jitsi/jitsi-meet/blob/4cc4c25691c4275919aed5cb23110ef4720519e0/react/features/mobile/navigation/components/settings/components/SettingsNavigationContainer.tsx#L38

Reply:
Ah kewl!